tensionmap.py

```diff
@@ -153,7 +153,7 @@ def tm_update(obj, context):
 
     # create vertex color list for faster access only if vertex color is activated
     if obj.data.tm_enable_vertex_colors:
-        vertex_colors = [0.0] * (number_of_tm_channels * num_vertices)
+        vertex_colors = [[0.0]*number_of_tm_channels]* num_vertices
```

Comment:
I would put spaces around the `*` operators. For consistency.

Reply:
Done!